styles/main.css

```diff
@@ -1,433 +1,651 @@
+  --gray: #f6f9f9;
+  --blue: #1da1f2;
+  --border-color: rgba(128, 128, 128, 0.145);
+}
 
 /* --------------------------------*/
+.font-chirp {
+  font-family: "chirp";
+}
+.h-100 {
+  height: 100%;
+}
+
+.w-100 {
+  width: 100%;
+}
 .twitter-color {
-    color :var(--blue);
-    fill: var(--blue);
+  color: var(--blue);
+  fill: var(--blue);
 }
 
 .gray {
-    color : gray
+  color: gray;
 }
 .il-b {
-    display: inline-block
+  display: inline-block;
+}
+.b-0 {
+  border: none;
 }
 .b-light {
-    border: 2px solid var(--border-color);
+  border: 2px solid var(--border-color);
 }
 
 .b-thin {
-    border: 1px solid var(--border-color);
+  border: 1px solid var(--border-color);
 }
 .b-b-thin {
-   
-    border-bottom: 1px solid var(--border-color);
+  border-bottom: 1px solid var(--border-color);
 }
 .b-sd-thin {
-    border-left: 1px solid var(--border-color);
-    border-right: 1px solid var(--border-color);
+  border-left: 1px solid var(--border-color);
+  border-right: 1px solid var(--border-color);
 }
 
 .b-eg-thin {
-    border-top: 1px solid var(--border-color);
-    border-bottom: 1px solid var(--border-color);
+  border-top: 1px solid var(--border-color);
+  border-bottom: 1px solid var(--border-color);
 }
 
 .b-white {
-    border-color: white
+  border-color: white;
 }
 
 .b-r-sm {
-    border-radius: 20%;
+  border-radius: 20%;
 }
 
-
 .b-r-md {
-    border-radius: 30px;
+  border-radius: 30px;
 }
 
 .b-r-sm {
-    border-radius: 15px;
+  border-radius: 15px;
 }
 .pd-0 {
-    padding:0;
+  padding: 0;
 }
 
 .pd-sm {
-    padding: 10px
+  padding: 10px;
 }
 
 .pd-md {
-    padding: 15px;
+  padding: 15px;
 }
 
 .pd-lg {
-    padding: 20px;
+  padding: 20px;
 }
 
 .pd-sd-sm {
-    padding-left:  10px;
-    padding-right:  10px;
-
+  padding-left: 10px;
+  padding-right: 10px;
 }
 .pd-sd-md {
-    padding-left:  15px;
-    padding-right:  15px;
+  padding-left: 15px;
+  padding-right: 15px;
+}
+
+.pd-eg-md {
+  padding-top: 15px;
+  padding-bottom: 15px;
 }
 
 .pd-sd-lg {
-    padding-left: 20px;
-    padding-right: 20px;
+  padding-left: 20px;
+  padding-right: 20px;
 }
 .mg-sd-sm {
-    margin-left: 10px;
-    margin-right: 10px;
+  margin-left: 10px;
+  margin-right: 10px;
 }
 
 .mg-sm {
-    margin: 10px;
+  margin: 10px;
 }
 
 .mg-eg-md {
-    margin-top: 15px;
-    margin-bottom: 15px;
+  margin-top: 15px;
+  margin-bottom: 15px;
 }
 
 .mg-eg-sm {
-    margin-top: 10px;
-    margin-bottom: 10px;
+  margin-top: 10px;
+  margin-bottom: 10px;
 }
 .mg-eg-xd {
-    margin-top: 5px;
-    margin-bottom: 5px;
+  margin-top: 5px;
+  margin-bottom: 5px;
 }
 .mg-eg-xs {
-    margin-top: 5px;
-    margin-bottom: 5px;
+  margin-top: 5px;
+  margin-bottom: 5px;
 }
 
 .mg-t-sm {
-    margin-top: 10px;
+  margin-top: 10px;
 }
 .mg-t-md {
-    margin-top: 15px;
+  margin-top: 15px;
 }
 .mg-0 {
-    margin: 0;
+  margin: 0;
 }
 
 a {
-    color: inherit;
-    text-decoration: none;
+  color: inherit;
+  text-decoration: none;
+}
+.z-1 {
```

Comment:
Isn't there a way to not use z-index ?

Reply:
https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Positioning/Understanding_z_index/Stacking_without_z-index    |    this helped me out when I had the same question